docs/Tutorials and concepts/Concepts/Out-parameter and its purpose.md

```diff
@@ -1,11 +1,11 @@
 In ImageJS several functions have an optional parameter called `out`.
-This parameter allows to choose which image to use as an output. It enables better memory management by allowing cloning image into an existing one instead of cloning it every time a function is executed.
+This parameter allows to choose which image to use as an output. It enables better memory management by allowing writing image into an existing one instead of cloning it every time a function is executed.
```

Comment:
```suggestion
You can use this parameter to select the image you want to use as the output. It helps with memory management by allowing you to write the image into an existing one instead of creating a new copy every time the function is run.
```